### ISSUE TITLE
Alternate view of period list, through a settings button as for spatial coverage?

### ISSUE THREAD

**Issue by atomrab** [periodo/periodo-client#296]
At the risk of violating the "if it ain't broke, don't fix it" principle, I've been wondering for a while about how to deal with the tension between original (idiosyncratic) source statements, and the normalized English labels and ISO dates when one is browsing the period list. How difficult would it be to add an alternate view, accessed through a settings button as Patrick did for the spatial coverage, where you have English-language label, ISO dates, and Wikidata spatial coverage? That could be the "parsed" or "normalized" view, and the current view (which uses primary label, original date assertions, and original spatial description) could be the "source" or "original" view.

I was just imagining getting a whole bunch of periods in Arabic script with AH dates, which would make it pretty hard for non-specialist users to navigate. Having an alternate fully parsed view would solve this problem immediately (plus the ones I've gone back and forth with for the dates -- I think I originally wanted not to show the ISO dates because then you lose the sense of the dating system).

**Comment by ptgolden** [comment]
Good idea. I'll give it a good think on Wednesday when I have some free time.

**Comment by rybesh** [comment]
I like this idea, and it would emphasize in the interface a design principle we've been at pains to maintain in the data.
